packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummaryResult.tsx

```diff
@@ -0,0 +1,103 @@
+import graphql from 'babel-plugin-relay/macro'
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+import {FONT_FAMILY} from 'parabol-client/styles/typographyV2'
+import {WholeMeetingSummaryResult_meeting$key} from 'parabol-client/__generated__/WholeMeetingSummaryResult_meeting.graphql'
+import React, {useEffect} from 'react'
+import {useFragment} from 'react-relay'
+import useAtmosphere from '../../../../../hooks/useAtmosphere'
+import SendClientSegmentEventMutation from '../../../../../mutations/SendClientSegmentEventMutation'
+import {AIExplainer} from '../../../../../types/constEnums'
+import {isNotNull} from '../../../../../utils/predicates'
+import EmailBorderBottom from './EmailBorderBottom'
+
+const topicTitleStyle = {
+  color: PALETTE.SLATE_700,
+  fontFamily: FONT_FAMILY.SANS_SERIF,
+  fontWeight: 600,
+  lineHeight: '22px',
+  fontSize: 14,
+  padding: '8px 48px'
+}
+
+const explainerStyle = {
+  color: PALETTE.SLATE_700,
+  fontFamily: FONT_FAMILY.SANS_SERIF,
+  fontStyle: 'italic',
+  padding: '8px 48px',
+  fontSize: 14
+}
+
+const textStyle = {
+  color: PALETTE.SLATE_700,
+  fontFamily: FONT_FAMILY.SANS_SERIF,
+  padding: '0px 48px 8px 48px',
+  fontSize: 14
+}
+
+interface Props {
+  meetingRef: WholeMeetingSummaryResult_meeting$key
+}
+
+const WholeMeetingSummaryResult = (props: Props) => {
+  const {meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment WholeMeetingSummaryResult_meeting on RetrospectiveMeeting {
+        __typename
+        id
+        summary
+        phases {
+          phaseType
+          ... on DiscussPhase {
+            stages {
+              discussion {
+                summary
+              }
+            }
+          }
+        }
+        team {
+          tier
+        }
+      }
+    `,
+    meetingRef
+  )
+  const atmosphere = useAtmosphere()
+  const {summary: wholeMeetingSummary, team, phases} = meeting
+  const discussPhase = phases.find((phase) => phase.phaseType === 'discuss')
+  const {stages} = discussPhase ?? {}
+  const explainerText = team?.tier === 'starter' ? AIExplainer.STARTER : AIExplainer.PREMIUM_MEETING
+  useEffect(() => {
+    const discussionSummaries = stages?.map((stage) => stage.discussion?.summary).filter(isNotNull)
+    SendClientSegmentEventMutation(atmosphere, 'AI Summary Viewed', {
+      source: 'Meeting Summary',
+      tier: meeting.team.tier,
+      meetingId: meeting.id,
+      discussionSummaries,
+      summary: wholeMeetingSummary
```

Comment:
I thought we could add the summaries to the segment events. This way, we can read through the summaries to see if they make sense or if the OpenAI prompt needs adjusting. 

In the meeting summary, I'm tracking both the `wholeMeetingSummary` and all of the `discussionSummaries`, i.e. the summary of the discussion threads. The topic summary is being tracked in the discussion thread.

Reply:
-1 I'm not sure we really need to send the full summaries to the Segment events, and should generally try to avoid sending possible PII.

If we're storing them as normal discussion items in the DB, maybe we can just pass the ID into Segment?  We can still do the deeper analysis, without worrying about how this propagates through our tools.

Reply:
That makes sense! Updated 👍